vyxal/transpile.py

```diff
@@ -0,0 +1,164 @@
+"""
+File: transpile_ast.py
+Description: This module is for transpiling Vyxal to Python
+"""
+
+import secrets
+from typing import List, Union
+
+from vyxal import elements, helpers, lexer, parse
+from vyxal.lexer import Token, TokenType
+from vyxal.structure import *
+
+
+def lambda_wrap(branch: List[Structure]) -> Lambda:
+    """
+    Turns a List of structures into a single lambda  Useful
+    for dealing with the functions of modifiers. Note that single
+    elements pass their arity on to the lambda
+
+    Para
```

Comment:
??

Reply:
I think lyxal started writing "Parameters:" and didn't finish (or maybe I screwed up somehow?)